pipeline/experiment.py

```diff
@@ -16,10 +16,20 @@ class Session(dj.Manual):
     session_date: date
     session_time: time  # computer time for t=0. This is the start of the data acquisition from the master device (typically with the highest sampling rate, e.g., ephys acquisition device)
     unique index (subject_id, session_date, session_time)
+    session_basename: varchar(24) # unique id used for referencing session files. All files associated to that session should contain that base name (or be within a folder that does)
     -> lab.Person
     -> lab.Rig
     """
 
+    class Notes(dj.Part):
+        definition = """
+        # Notes for the recording session, to be entered manually or parsed from notes spreadsheet 
+        -> Session
```

Comment:
you can use `-> master` here to refer to the `Session` table, since this `Notes` table is a part-table of `Session`